 connman (1.27-netgear1) unstable; urgency=low
 .
   * Treat bond interfaces as ethernet on recent kernels.
Author: Justin Maggard <justin.maggard@netgear.com>

---

--- connman-1.27.orig/src/rtnl.c
+++ connman-1.27/src/rtnl.c
@@ -170,6 +170,9 @@ static void read_uevent(struct interface
 		} else if (strcmp(line + 8, "gadget") == 0) {
 			interface->service_type = CONNMAN_SERVICE_TYPE_GADGET;
 			interface->device_type = CONNMAN_DEVICE_TYPE_GADGET;
+		} else if (strcmp(line + 8, "bond") == 0) {
+			interface->service_type = CONNMAN_SERVICE_TYPE_ETHERNET;
+			interface->device_type = CONNMAN_DEVICE_TYPE_ETHERNET;
 
 		} else {
 			interface->service_type = CONNMAN_SERVICE_TYPE_UNKNOWN;
